src/views/labs/LabView/LabView.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <EternaPage v-if="data" title="Lab Details">
+  <EternaPage v-if="lab" title="Lab Details">
```

Comment:
I think I prefer `data`... But I'll have to think about it more

Reply:
We can't use `data` with a computed property - it gets clobbered (per our discussion the other day). I'm also not sure why it's particularly helpful to make the differentiation?

Reply:
My thought is that it's part of the template for pages with a query - only show the content if your query is done, regardless of its content.

Reply:
If so, it would be `pageData`, but I don't know whether that level of consistency is really necessary... in the future we'll ideally be using graphql anyways so you won't have one consistent high-level property of the same name since you define the things you're requesting directly on the instance

Reply:
I'll see as I write more pages. At worst I'll revert it to pageData, but for now I'll keep your changes.